api/controllers/file_processor/file-base64-upload.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * file_processor/file-base64-upload.js
+ *
+ *
+ * @api {post} /file/upload/base64/:objID/:fieldID Upload
+ * @apiParam {string: base64} file
+ * @apiParam {uuid} photoID
+ * @apiParam {uuid} objID
+ * @apiParam {uuid} fieldID
+ * @apiParam {string} uploadedBy
+ * @apiParam {number} size
+ * @apiParam {string} type
+ * @apiParam {string} fileName
+ * @apiUse successRes
+ * @apiSuccess (200) {Object} data
+ * @apiSuccess (200) {string} data.uuid
+ */
+
+const async = require("async");
+const { error } = require("console");
+
+var inputParams = {
+   file: { string: true, required: true },
+   photoID: { string: true, required: true },
+   objID: { string: true, required: true },
+   fieldID: { string: true, required: true },
+   uploadedBy: { string: true, required: true },
+   size: { number: true, required: true },
+   type: { string: true, required: true },
+   fileName: { string: true, required: true },
+};
+
+module.exports = function (req, res) {
+   // Package the Find Request and pass it off to the service
+   req.ab.log(`file_processor::file-base64-upload`);
+
+   // verify your inputs are correct:
+   if (
+      !(req.ab.validUser(/* false */)) ||
+      !req.ab.validateParameters(inputParams /*, false , valuesToCheck*/)
+   ) {
+      req.ab.log("... base64Params invalid");
+      req.ab.log(error);
```

Comment:
`error` here refers to `console.error` defined on line 20? So we're logging the function?
Also `validateParameters` should have its own error handling. Not sure logging here is necessary.